# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../eina/hotfix.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- eina/eina-0.pc.in.vanilla	2008-10-27 15:31:51.000000000 +0100
+++ eina/eina-0.pc.in	2008-10-27 15:31:58.000000000 +0100
@@ -7,6 +7,6 @@
 Description: A Library that implements fast data types and miscellaneous tools
 Requires:
 Version: @VERSION@
-Libs: -L${exec_prefix}/lib -leina
+Libs: -L${libdir} -leina
 Libs.private: -ldl
 Cflags: -I${prefix}/include/eina-@VMAJ@ -I${prefix}/include/eina-@VMAJ@/eina
